content/actions/guides/building-and-testing-python.md

```diff
@@ -421,14 +427,23 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+          pip install build
+      - name: Build package
         run: |
-          python setup.py sdist bdist_wheel
-          twine upload dist/*
+          python -m build
```

Comment:
For a one-line script, you don't need the `|`:

```
run: python -m build
```

Reply:
Thanks, I've updated this now 👍